src/drivers/mct/cime_config/config_component.xml

```diff
@@ -1949,6 +1949,18 @@
     <desc>ROOTPE (mpi task in MPI_COMM_WORLD) for each component</desc>
   </entry>
 
+  <entry id="MULTI_DRIVER">
+    <type>logical</type>
+    <default_value>FALSE</default_value>
+    <valid_values>TRUE,FALSE</valid_values>
+    <group>mach_pes</group>
+    <file>env_mach_pes.xml</file>
+    <desc>MULTI_DRIVER mode provides a separate driver (and coupler) component for each
```

Comment:
I would rephrase this line as:
<desc>MULTI_DRIVER mode provides a separate driver (and coupler component) for each